modules/ROOT/pages/kubernetes/index.adoc

```diff
@@ -15,6 +15,7 @@ This chapter describes the following:
 * xref:kubernetes/helm-charts-setup.adoc[Configure the Neo4j Helm chart repository] -- Configure the Neo4j Helm chart repository and check for the available charts.
 * xref:kubernetes/quickstart-standalone/index.adoc[Quickstart: Deploy a standalone instance] -- Deploy a Neo4j standalone instance to a cloud (GKE, AWS, AKS) or a local (via Docker Desktop for macOS) Kubernetes cluster.
 * xref:kubernetes/quickstart-cluster/index.adoc[Quickstart: Deploy a cluster] -- Deploy a Neo4j cluster to a cloud (GKE, AWS, AKS) Kubernetes cluster.
+* xref:kubernetes/quickstart-analytics-cluster.adoc[Quickstart: Deploy a Neo4j cluster for analytic queries] -- Deploy an analytics Neo4j cluster to a local or a cloud (GKE, AWS, AKS) Kubernetes cluster.
```

Comment:
it would be good to be specific in the title "Deploy a Neo4j Cluster for analytics (1 primary + N Secondary)" since N primary and N secondary was already possible

Reply:
I thought that we could expand this page similar to how this topic is presented in the [Clustering chapter ](https://neo4j.com/docs/operations-manual/current/clustering/setup/analytics-cluster/), and have two sections an analytics cluster with fault-tolerance and without.